### ISSUE TITLE
UI Narration

### ISSUE THREAD

**Issue by Pinsplash** [Pinsplash/magma-source#167]
This will be a big one and i'll have to look into it a lot more. can hl2 work with this out of the box?

**Comment by conneath** [comment]
this might require editing of GameUI, which does not have its source included in source sdk 2013
